token.go

```diff
@@ -178,3 +179,32 @@ func (m TokenModule) postTokenHandler(c *gin.Context) {
 
 	c.JSON(http.StatusCreated, token)
 }
+
+// postTokenHandler godoc
+// @summary Put token in database.
+// @description Put token in database. Provider in post object has to exists or should be empty.
+// @description Provider will has to be updated token ID during this operation.
+// @tags token
+// @accept json
+// @produce json
+// @param token body TokenWithProviderID _ "token object"
+// @success 200 {object} Token
+// @failure 400 {object} problem.Response "Bad request"
+// @failure 401 {object} problem.Response "Unauthorized or missing jwt token"
+// @failure 502 {object} problem.Response "Database is unreachable"
+// @router /token [put]
+func (m TokenModule) putTokenHandler(c *gin.Context) {
+	var token Token
+	if err := c.ShouldBindJSON(&token); err != nil {
+		problem.WriteInvalidBindError(c, err, "One or more parameters failed to parse when reading the request body.")
+		return
+	}
+	var placedToken Token
```

Comment:
Same comment here about the naming. Suggest naming them equivalent to inside the `putProviderHandler`

- `token` &rarr; `inputToken`
- `placedToken` &rarr; `token`

Reply:
Taken as refactor commit where `token → inputToken` & `placedToken → token`